### DISCUSSION TITLE
Oqtane for Dummies

### DISCUSSION THREAD

**Discussion by BGillham** [oqtane/oqtane.framework#1334] [Q&A]
Is there an "Oqtane for Dummies" type of blog for those who are experienced with DNN to explain equivalents like Portal Setup and Roles etc? I am brand-new and come from DNN background and like the initial look of Oqtane so I would like to know if there is some type of quick-start Blog / Documentation...

**Comment by sbwalker** [comment]
Have you seen the documentation:

https://docs.oqtane.org/admin/index.html
